boards/arm/efr32bg_sltb010a/doc/index.rst

```diff
@@ -0,0 +1,184 @@
+.. _efr32bg_sltb010a:
+
+EFR32BG-SLTB010A
+################
+
+Overview
+********
+
+The EFR32™ Blue Gecko Starter Kit EFR32BG-SLTB010A (a.k.a Thunderboard EFR32BG22)
+contains a MCU from the EFR32BG family built on ARM® Cortex®-M33F
+processor with low power capabilities.
+
+.. image:: ./efr32bg_sltb010a.jpg
+   :align: center
+   :alt: EFR32BG-SLTB010A
+
+Hardware
+********
```

Comment:
The number of '*' must match the length of the word.

Reply:
It is matching I guess. Am I missing something?

Reply:
Looks not matching. Maybe just formatting issue from my app. Take a look on ci output. There are many mistakes.

Reply:
could you please help me how to check CI output?

Reply:
The length of the header should be correct. Both line 17 and line 18 contain 8 characters.

Reply:
Done